includes/emails/class-give-email-tags.php

```diff
@@ -1223,3 +1230,78 @@ function __give_20_bc_str_type_email_tag_param( $tag_args ) {
 
 	return $tag_args;
 }
+
+/**
+ * Email template tag: {reset_password_link}
+ *
+ * @param array $tag_args Array of arguments for email tags.
+ *
+ * @since 2.0
+ *
+ * @return array
+ */
+function give_email_tag_reset_password_link( $tag_args ) {
+
+	$reset_password_url = give_get_reset_password_url(
```

Comment:
@mehul0810 Add support for donor id here. check how I am using switch case in email tag render functions.